charts/licensify/templates/deployment.yaml

```diff
@@ -61,6 +64,8 @@ spec:
               value: do-not-use
             - name: GDS_CONFIG_FILE
               value: /etc/licensing/config.properties
+            - name: LOGGING_CONFIG_FILE
+              value: ../../../app/conf/logging.xml
```

Comment:
optional: is there a benefit to using this odd-looking relative path here? (as opposed to just having it match `/app/conf/logging.xml` below?

Reply:
Ah no, it's just that what ever loads the config always assumes its relative even with a starting "/", so setting "/app/conf/logging.xml" ends up being something like "/app/feed/bin/app/conf/logging.xml".

Reply:
Haha oh wow. No worries! (thanks for clearing that up!)